packages/notifications/src/elements/Alert.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledAlert, IStyledAlertProps } from '../styled';
+import { VALIDATION } from '../utils/types';
+
+/**
+ * Supports all `<div>` props
+ */
+export const Alert = React.forwardRef<HTMLDivElement, IStyledAlertProps>((props, ref) => (
```

Comment:
These components are able to consume our custom props from `IStyledAlertProps`, but they're also able to consume all valid HTMLAttributes for a DIV element.

A more accurate type definition would we `IStyledAlertProps & HTMLAttributes<HTMLDivElement>`

Reply:
This should be replicated across all `forwardRef` usages

Reply:
Nice catch, I've updated the type definition for the generic to accept `HTMLAttributes<HTMLDivElement>`.